Core/Extensions/CollectionExtensions.cs

```diff
@@ -9,12 +9,22 @@ public static class CollectionExtensions
     {
         public static void CopyTo<T>(this IEnumerable<T> sourceCollection, ICollection<T> targetCollection)
         {
+            if (sourceCollection is null)
```

Comment:
Do you really prefer `is` over `==` here?

Reply:
It's now inconsistent in the code, something there is a `is null` and `== null` check in the same method.

Reply:
`is` is generated by the quick fix and it's not wrong... so seemed easiest to use it.